.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+name: Build
+
+on:
```

Comment:
Can you add a `workflow_dispatch` event to all these to manually trigger them on branches?

Reply:
What do you mean by "all of these"? All workflows? Or just test and build workflows?

Reply:
Everywhere would be nice. But at least in the two that you touch here.